pkg/movingaverage/moving_average_test.go

```diff
@@ -81,3 +82,33 @@ func (t *testMovingAvg) TestMedianFilter(c *C) {
 	t.checkAdd(c, mf, data, expected)
 	t.checkSet(c, mf, data, expected)
 }
+
+type testCase struct {
+	ma       MovingAvg
+	expected []float64
+}
+
+func (t *testMovingAvg) TestMovingAvg(c *C) {
+	var empty float64 = 0
+	data := []float64{120, 130, 140, 150, 145, 136, 121, 132, 145, 156, 148, 157, 175}
```

Comment:
This code may be hard to maintain in the later just like the tests of the hot region